docs/releases/0_9_10.rst

```diff
@@ -168,7 +168,7 @@ Carbon
 * Only log Carbon queue fullness once (sidnei)
 * Only log Carbon queue space free if it was once full (sidnei)
 * Log a message with the affected filename when a Whisper update fails (bmhatfield)
-* Move carbon instance logs to their own own directory to prevent clobbering
+* Move carbon instance logs to their own directory to prevent clobbering
```

Comment:
I'm still experimenting w/ duplicate word correction, but I'm pretty sure this is right.

Note that some projects don't like changing old release notes. Happy to drop...